### ISSUE TITLE
Add a CMake linter GitHub action

### ISSUE THREAD

**Issue by CuriousGeorgiy** [tarantool/tarantool#9224]
Let's add a [CMake-format lint](https://github.com/marketplace/actions/cmake-format-lint-action) action to maintain the code style of our CMake scripts.

**Comment by CuriousGeorgiy** [comment]
@igormunkin is it so important whether a tool like this is supported? I mean we don't have really high expectations for CMake, i.e., we just need to check simple stuff like identation, keyword register, brackets, line length, etc. Of course, it would be helpful to  So I believe we can try this tool out, see if it fits our needs, and go for using it

**Comment by igormunkin** [comment]
> is it so important whether a tool like this is supported?
It definitely is, if there is a supported and better alternative :)

I'm not against using this particular tool for now, but very curious whether there are better alternatives? If you didn't find them, it's OK, just proceed with the tool you chose.

**Comment by CuriousGeorgiy** [comment]
@igormunkin

> It definitely is, if there is a supported and better alternative :)

Yeah, I implied that there are no better alternatives (I also didn't find any)

**Comment by ligurio** [comment]
There is an alternative solution:
- implement CMake language support in semgrep [^1]
- create semgrep rules and run them in a project

[^1]: https://github.com/semgrep/semgrep/issues/10000
